docs/en/modules/wifi.md

```diff
@@ -1,4 +1,8 @@
 # WiFi Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2015-05-12 | [dnc40085](https://github.com/dnc40085) | [dnc40085](https://github.com/dnc40085) | [wifi.c](../../../app/modules/wifi.c)|
```

Comment:
I believe it was funshine that originally wrote the wifi module. I wouldn't mind taking the credit for writing it though :grinning: